src/Backend/Version.ts

```diff
@@ -32,7 +32,13 @@ class Version {
     return ret;
   }
 
-  // TODO: Add operator <, <=, ==, >, >=
+  equals(ver: Version): Boolean {
```

Comment:
(optional) I'm not sure about this but there is `Boolean` Object and `boolean` primitive type. I think return type would be `boolean`

Reply:
FYI) https://stackoverflow.com/questions/64443288/what-is-the-difference-between-boolean-and-boolean-in-typescript

Reply:
Thanks!

Reply:
Interesting.
```ts
if (new Boolean(false)) {
    console.log('run only when if true');
}
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean
```
var x = Boolean(expression);     // use this...
var x = !!(expression);          // ...or this
var x = new Boolean(expression); // don't use this!
```